sopel/irc/__init__.py

```diff
@@ -358,6 +359,7 @@ def on_message_sent(self, raw: str) -> None:
                 ":{0}!{1}@{2} {3}".format(self.nick, self.user, host, raw),
                 url_schemes=self.settings.core.auto_url_schemes,
                 identifier_factory=self.make_identifier,
+                statusmsg_prefixes=self._isupport.get('STATUSMSG'),
```

Comment:
Same question as above. Previously existing uses of ISUPPORT data in this file use the property, not the attr directly.

```suggestion
                statusmsg_prefixes=self.isupport.get('STATUSMSG'),
```

Reply:
I wonder if the other usage are there because it was made to be backward compatible? I honestly can't tell, it's been too long.